Makefile

```diff
@@ -287,9 +286,16 @@ check-apply-migration:
 #############################
 
 # build devcontainer locally
+.PHONY: build-devcontainer
 build-devcontainer:
 	devcontainer build --workspace-folder=.github --push=false --image-name="ghcr.io/bucketeer-io/bucketeer-devcontainer:latest"
 
+.PHONY: push-devcontainer
+push-devcontainer:
+	@echo $(PAT) | docker login ghcr.io -u $(GITHUB_USER_NAME) --password-stdin
+	docker tag ghcr.io/bucketeer-io/bucketeer-devcontainer:latest ghcr.io/bucketeer-io/bucketeer-devcontainer:latest
+	docker push ghcr.io/bucketeer-io/bucketeer-devcontainer:latest
+
```

Comment:
Do we need to push the image?
We can run a container that is locally built.

Reply:
I have removed it.